tests/test_syslog.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+
+Documentation       This suite is for testing syslog function of Open BMC.
+
+Resource            ../lib/rest_client.robot
+Resource            ../lib/utils.robot
+
+*** Variables ***
+${IP_ADDRESS}              9.109.116.67
```

Comment:
you should not hard code an IP in the testcase it self.  It is point less because the subnet where I am testing is not where someone else is testing.  I would suggest code that either...
1) is told the new ip from a the tox.ini global variable
2) is calculated based on the ip of the victim